tests/func/test_metrics.py

```diff
@@ -711,7 +713,7 @@ def _test_metrics(self, func):
         )
 
         res = self.dvc.metrics.show(
-            "", all_branches=True, typ="json", xpath="metrics"
+            [], all_branches=True, typ="json", xpath="metrics"
```

Comment:
I guess we could simply
```suggestion
            all_branches=True, typ="json", xpath="metrics"
```
here :slightly_smiling_face: